doc/en/user/source/extensions/controlflow/index.rst

```diff
@@ -179,3 +186,48 @@ Assuming the server we want to protect has 4 cores a sample configuration could
+    19 018 15:18:31 'http-nio-8080-exec-8' DEBUG  [flow.controller] - X-Forwarded-For: 185.230.235.34, 10.12.81.32 -> 185.230.235.34
+    19 018 15:18:44 'http-nio-8080-exec-8' DEBUG  [flow.controller] - IpFlowController(3) 185.230.235.34, concurrent requests: 3
+    19 018 15:18:44 'http-nio-8080-exec-8' DEBUG  [flow.controller] - IpFlowController(3,185.230.235.34) queue size 3
+    19 018 15:18:44 'http-nio-8080-exec-8' DEBUG  [geoserver.flow] - Request [WFS 1.0.0 GetFeature] exit  IpFlowController(3)
+    19 018 15:18:44 'http-nio-8080-exec-8' DEBUG  [geoserver.flow] - Request [WFS 1.0.0 GetFeature] enter BasicOWSController(wps.execute,SimpleBlocker(4))/0
+    19 018 15:18:44 'http-nio-8080-exec-8' DEBUG  [geoserver.flow] - Request [WFS 1.0.0 GetFeature] exit  BasicOWSController(wps.execute,SimpleBlocker(4))/0
+    19 018 15:18:44 'http-nio-8080-exec-8' DEBUG  [geoserver.flow] - Request [WFS 1.0.0 GetFeature] enter BasicOWSController(wms.getmap,SimpleBlocker(6))/0
+    19 018 15:18:44 'http-nio-8080-exec-8' DEBUG  [geoserver.flow] - Request [WFS 1.0.0 GetFeature] exit  BasicOWSController(wms.getmap,SimpleBlocker(6))/0
+    19 018 15:18:44 'http-nio-8080-exec-8' DEBUG  [geoserver.flow] - Request [WFS 1.0.0 GetFeature] enter BasicOWSController(wfs.getfeature,SimpleBlocker(8))/2
+    19 018 15:18:44 'http-nio-8080-exec-8' DEBUG  [geoserver.flow] - Request [WFS 1.0.0 GetFeature] exit  BasicOWSController(wfs.getfeature,SimpleBlocker(8))/3
+    19 018 15:18:44 'http-nio-8080-exec-8' DEBUG  [geoserver.flow] - Request [WFS 1.0.0 GetFeature] enter GlobalFlowController(SimpleBlocker(50))/2
+    19 018 15:18:44 'http-nio-8080-exec-8' DEBUG  [geoserver.flow] - Request [WFS 1.0.0 GetFeature] exit  GlobalFlowController(SimpleBlocker(50))/3
+    19 018 15:18:44 'http-nio-8080-exec-8' INFO   [geoserver.flow] - Request control-flow performed, running requests: 3, blocked requests: 12
+
+In this example:
+
+* A RateFlowController imposes a delay of 10 seconds on WPS andm WMS requests requests.
+* A WCS concurrency control has a limit of 1, but there are 0 WCS request executing.
+* A concurrency limit by IP address is set to 3, and there are 3 requests executing from a single IP (with details on how the IP address has been extracted from the X-Forwarded-For header).
+* A concurrency control on wps.execute with 4 requests is in place, with none executing.
+* A concurrency control on wms.getmap with 6 requests is in place, with none executing.
+* A concurrency control on wfs.getfeature with 8 requests is in place, with 3 executing.
+* A global concurrency control with 50 requests is in place, with 3 executing.
+* In summary, 3 requests are executing, and 12 are blocked. From this situation, they are queued on the single IP flow controller.
+* Looking at the timings, this request has been delayed for about 13 seconds on the single IP flow controller.
```

Comment:
I'm not very familiar with Control Flow:
why all the logs reports WFS 1.0.0 GetFeature? Is it because those details are reported when evaluating the status during that WFS getRequest?

Reply:
It's the same request being processed through all the configured controllers.